3rd-party/class.jetpack-amp-support.php

```diff
@@ -45,6 +45,17 @@ static function init() {
 		add_filter( 'amp_post_template_metadata', array( 'Jetpack_AMP_Support', 'amp_post_template_metadata' ), 10, 2 );
 	}
 
+	/**
+	 * Disable theme features not supported by AMP.
+	 *
+	 * @uses remove_theme_support
+	 */
+	public function disable_theme_features() {
+		if ( self::is_amp_request() ) {
+			remove_theme_support( 'jetpack-devicepx' );
```

Comment:
I think the AMP module should be responsible for removing the devicepx support to ensure that API changes in the AMP module don't have to be updated across other modules.